src/MongoDB.Analyzer/Core/Poco/Data/Movie.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+    "movie": [
```

Comment:
We also need Id, Number (for CustomerNumber), Phone, Date, DOB.

Reply:
Done